From 5fad7c6b65ff4cf39cdd0ef381f5ed4f5276d406 Mon Sep 17 00:00:00 2001
From: Francois Gaffie <francois.gaffie@intel.com>
Date: Wed, 18 Jul 2012 18:34:50 +0200
Subject: Add parameter key definition for HAC

BZ: 47599
[CLEAN]

This patch adds both HAC key define and HAC value pairs define.
HAC stands for Hearing Aid Compatible.

Change-Id: Iace10de2f563291c74a64b1288733625b4a1a928
Orig-Change-Id: I559643085412a11dcaff882cf72dc11ebb6ec693
Category: device enablement
Domain: AudioComms-Common
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Francois Gaffie <francois.gaffie@intel.com>
Signed-off-by: Renaud Viry <renaudx.viry@intel.com>
---
 include/hardware/audio.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index f3dfc5e..f0e5744 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -99,6 +99,11 @@ __BEGIN_DECLS
 #define AUDIO_PARAMETER_VALUE_TTY_HCO "tty_hco"
 #define AUDIO_PARAMETER_VALUE_TTY_FULL "tty_full"
 
+/* HAC device selection */
+#define AUDIO_PARAMETER_KEY_HAC_SETTING "HACSetting"
+#define AUDIO_PARAMETER_VALUE_HAC_ON "ON"
+#define AUDIO_PARAMETER_VALUE_HAC_OFF "OFF"
+
 /* A2DP sink address set by framework */
 #define AUDIO_PARAMETER_A2DP_SINK_ADDRESS "a2dp_sink_address"
 
-- 
1.7.4.1

